benchmarks/DeepLearning/Ops/BatchMatMulOp/CMakeLists.txt

```diff
@@ -51,6 +40,50 @@ add_custom_command(OUTPUT batch_matmul_auto_vectorization.o
 add_library(BatchMatMulAutoVectorization STATIC batch_matmul_auto_vectorization.o)
 set_target_properties(BatchMatMulAutoVectorization PROPERTIES LINKER_LANGUAGE CXX)
 
+# BatchMatMul Vectorization
```

Comment:
Since we have implemented the related pass for optimization, can we use it here? That is, replace the source code with linalg operation and ultilize the pass to validate e2e lowering.

Reply:
The CMakeLists have been updated for end-to-end (e2e) lowering.